docs/modules/ROOT/pages/documentation/payara-server/logging/fast-logging.adoc

```diff
@@ -0,0 +1,27 @@
+[[fast-logging]]
+= Fast Logging
+
+By default, a toString() is forced on all parameters within the same thread as the application. This stops an immutable object being changed after the log statement is issued but can cause a performance impact with JPA entities as it may result in database access.
```

Comment:
>This stops an immutable object being changed after the log statement is issued but can cause a performance impact with JPA entities as it may result in database access.

Should be reworded as not entirely correct

This stops a **mutable** object being changed after the log statement is issued **and thus wrong values be written out in the log entry** but can cause a performance impact with JPA entities as it may result in database access.